packages/ses/types.d.ts

```diff
@@ -44,7 +44,7 @@ export interface RepairOptions {
     | 'safeEval'
     | 'unsafeEval'
     | 'noEval';
-  stackFiltering?: 'concise' | 'verbose';
+  stackFiltering?: 'concise' | 'omit-frames' | 'shorten-paths' | 'verbose';
```

Comment:
idea:

```ts
export type StackFilter = 'concise' | 'omit-frames' | 'shorten-paths' | 'verbose';
```

then reference here and [here](https://github.com/endojs/endo/pull/2747/files#diff-6f0131b09b82217195df81c38e36c57ed94ebfa022aa6421c5770dd05fefe457R208)

Reply:
Would break the general pattern followed by all the other options. I added a TODO explaining what a fix would need to do:

```js
// TODO Somehow remove the redundancy between these type deinitions and the
// inline casts on each call to `getenv` in `lockdown.js`. Hopefully we can
// keep the type info in those casts so it is easily comparable by eye to
// the parameters of that call to `genev`.
```

Resolving.